example/provider-extensions/garden/project/credentials/infrastructure-secrets.yaml.tmpl

```diff
@@ -0,0 +1,34 @@
+##########
```

Comment:
Same here, instead of duplicating this information in g/g, users should rather refer to the docs of the provider extension of their choice.